python/utils.py

```diff
@@ -198,17 +194,18 @@ def qgis_excepthook(type, value, tb):
 
 
 def installErrorHook():
+    """
+    Installs the QGIS application error/warning hook
```

Comment:
what's the effect? (I.e. be specific about if it's worth calling this in a script environment ;) )